### ISSUE TITLE
nswag issue with .net 6

### ISSUE THREAD

**Issue by Poltergeisen** [jasontaylordev/CleanArchitecture#504]
**Describe the bug**
I'm receiving this message when building. It seems like nswag is not being detected/loaded correctly. Is it supposed to be looking for the net5.0 path?

```
C:\code\poc\src\WebUI\WebUI.csproj(59,9): error MSB3073: The command "dotnet "C:\Users\adamg\.nuget\packages\nswag.msbuild\13.9.4\build\../tools/Net50/dotnet-nswag.dll" run nswag.json /variables:Configuration=Debug" exited with code -2147450730.   
```

**To Reproduce**
1. Go to webui with a new .net 6.0 SDK install
2. run `dotnet build`

**Expected behavior**
The build should complete

**Comment by Poltergeisen** [comment]
Seems like it could be something to do with this?

https://github.com/RicoSuter/NSwag/wiki/Assembly-loading#net-core

**Comment by mrwsoftware** [comment]
I've created a pull request which I believe should address this - 

[https://github.com/jasontaylordev/CleanArchitecture/pull/507](url)

Hope it helps somebody.

**Comment by ageisen2000** [comment]
seems like this was merged but im still having issues with creating a brand new app. Any way to make the dotnet new command use this?

**Comment by haydenhancock** [comment]
Same issue here. I am wondering if this has something to do with the template itself (maybe this needs to be updated). I haven't found any documentation on how this template actually works so I can't provide any additional guidance.

**Comment by ageisen2000** [comment]
@haydenhancock I'm fairly confident @jasontaylordev needs to publish the nuget package based on master and this will be fixed

**Comment by vpkopylov** [comment]
@waki68 The issue can be solved by implementing changes from #507 manually in the source code of your project. It's quite easy

**Comment by jasontaylordev** [comment]
All fixed as per #507
